.travis.yml

```diff
@@ -9,67 +9,44 @@ stages:
 jobs:
   include:
     - name: Syntax and Tests
-      script:
-        - make install-src
-        - make fast
+      script: make check_notebooks install-src fast
       after_success:
         - bash <(curl -s https://codecov.io/bash)
     - name: Docker
-      if: branch =~ ^master OR branch =~ ^merge
-      script:
-        - make docker
+      if: type = push and branch =~ /^(master|merge)/
+      script: make docker
     - name: Allen Notebook
-      if: (branch =~ ^master OR branch =~ ^merge) AND tag != cron
-      env:
-      script:
-        - make install-src
-        - make run__notebooks/py/allen_smFISH.py
+      if: type = push and branch =~ /^(master|merge)/
+      script: make install-travis run__notebooks/py/allen_smFISH.py
```

Comment:
why not just do `make install-src <notebook>`?

Reply:
If the TRAVIS_EVENT_TYPE logic is also in the job definition, then things get duplicated.

Based on "_I would prefer that the make commands in .travis.yml invoke the explicit rule that it needs (install-src vs install-pypi)_", happy to expand that out, but the above becomes:

```
      if: type = push and branch =~ /^(master|merge)/ and type = cron
      script: make install-pypi run__notebooks/py/allen_smFISH.py
      if: type = push and branch =~ /^(master|merge)/ and type != cron
      script: make install-src run__notebooks/py/allen_smFISH.py
```

which felt error prone.

Reply:
Sorry, I take that back. In this case since `type = push` is already listed, yes, I can change from travis to the specific version. The other cases look more like what I pasted here. I used `install-travis` everywhere for consistency. To be clear:

 * You want specific targets where possible.
 * You'd like `install-travis` to be removed completely?

Reply:
Doesn't every stage have a if: type = xxx?  Or am I completely oblivious?

I think that `install-travis` makes things less clear, so I would like to get rid of that.  If there's a case for multiple targets to be invoked from one stage, pls let me know.

Reply:
Ditched it. Merge build is running in https://travis-ci.org/spacetx/starfish/builds/433298975